docs/configure-upstream-collector.md

```diff
@@ -0,0 +1,13 @@
+# Configure an upstream collector
+
+You can configure an upstream collector, like a [custom collector](https://opentelemetry.io/docs/collector/custom-collector/) or [contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib) distribution for the collector, to collect logs and metrics and send them to Elastic Observability.
+
+<!-- We might want to add some pros and cons of using an upstream collector -->
+
+For a more seamless experience, use the Elastic distribution for the OpenTelemetry collector.
+Refer to the [guided onboarding](guided-onboarding.md) docs or the [manual configuration](manual-configuration.md) docs for more on configuring the Elastic distribution.
+
+## Upstream collector configuration example
+Use the following configuration example as a reference when configuring an upstream collector.
+
+<!-- Add config example-->
```

Comment:
@mlunadia I asked in the OTel devs channel if there was an example we could use here, but haven't heard back. Do you have an idea of who might be able to help here?

Reply:
Example configurations for our distro are [here](https://github.com/elastic/elastic-agent/tree/main/internal/pkg/otel/samples)

For now I think this section should remain explaining at high level the components needed [to create a custom distro](https://opentelemetry.io/docs/collector/custom-collector/) that includes what we have in our distro.

Reply:
The configurations shouldn't be much different